### PR TITLE
Swagger change for Update configuration dynamic group

### PR DESCRIPTION
**Pull request by jemex**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by marstr — COMMENTED** [review]
A couple of nits, but otherwise nothing troublesome.

File: specification/automation/resource-manager/Microsoft.Automation/preview/2017-05-15-preview/softwareUpdateConfiguration.json [hunk 1]
```diff
@@ -750,6 +754,75 @@
...
+        "tagSettings": {
+          "type": "object",
+          "description": "tag filter information of the Vm.",
+          "$ref": "#/definitions/TagSettingsProperties"
+        }
+      }
+    },
+    "TagSettingsProperties": {
+      "type": "object",
+      "description": "Tag filter Information of the VM.",
```
[the remark this answers is not in the document]
Comment (jemex) [reply specification/automation/resource-manager/Microsoft.Automation/preview/2017-05-15-preview/softwareUpdateConfiguration.json]: Thank you, Fixed

**Review by marstr — APPROVED** [review]
The descriptions are still filled with grammatical errors and hurried capitalization. However, I cannot force you to appear professional in front of your customers. Please consider spending some real time trying to make this look nice.

**Comment by marstr** [comment]
Are you ready for me to merge, @jemex?

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Comment by marstr** [comment]
Thank you for making those updates, @jemex. I truly appreciate it.

Let me know when you're ready to merge.

**Comment by marstr** [comment]
I got word from @jemex offline that was safe to merge this PR.

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Pull request merged**